app/helpers/embed_player_helper.rb

```diff
@@ -93,6 +93,12 @@ def embed_player_category_options(podcast, selected)
     options_for_select(opts, selected)
   end
 
+  def enclosure_with_token(ep)
+    url = EnclosureUrlBuilder.new.base_enclosure_url(ep.podcast, ep, ep.podcast.default_feed)
```

Comment:
that's right, `base_enclosure_url` expands the template, but doesn't add in the prefix values, exactly.